orchagent/portsorch.h

```diff
@@ -57,6 +57,9 @@ class PortsOrch : public Orch, public Subject
     bool setHostIntfsOperStatus(sai_object_id_t id, bool up);
     void updateDbPortOperStatus(sai_object_id_t id, sai_port_oper_status_t status);
     bool bindAclTable(sai_object_id_t id, sai_object_id_t table_oid, sai_object_id_t &group_member_oid, acl_stage_type_t acl_stage = ACL_STAGE_INGRESS);
+
+    bool getPortPfc(sai_object_id_t portId, uint8_t *pfc_bitmask);
```

Comment:
In getPortPfc, it queries sai attribute each time which is very insufficient. Can we cache the value and let getPortPfc only obtain the cached value? The cached value is only updated every time value changes

Reply:
Implemented required changes